src/SideNav.js

```diff
@@ -15,11 +15,12 @@ class SideNav extends Component {
   }
 
   render() {
-    const { className, children, fixed, ...props } = this.props;
+    const { className, children, ...props } = this.props;
     delete props.id;
     delete props.trigger;
     delete props.options;
-
+    delete props.fixed;
```

Comment:
the way deconstruct works is that if you add `fixed` to L18 then you don't need to delete it from the `props` object

Reply:
also add trigger to L18 and delete it from L20